css/index.css

```diff
@@ -137,10 +139,40 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 0.875rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media (max-width: 1000px) {
+    .middle-content .boxes .box .box2 {
+        background-color: forestgreen;
+    }
+
+    .middle-content .boxes .box .box4 {
+        background-color: darkorchid;
+    }
+
+    .middle-content .boxes .box .box6 {
+        background-color: hotpink;
+    }
+
+    .middle-content .boxes .box .box8 {
+        background-color: indigo;
+    }
+
+    .middle-content .boxes .box .box10 {
```

Comment:
Also try removing the .box class from each of these, so for example you'll only have: ``` .middle-content .boxes box2 {
          background-color: forestgreen;
}```